index.js

```diff
@@ -0,0 +1,101 @@
+const { PrismaClient } = require('@prisma/client')
+const { v4: uuidv4 } = require('uuid')
+
+const prisma = new PrismaClient()
+
+// retryTxn wraps SQL operations in an explicit transaction.
+// If the transaction fails, the function retries the operation, with exponential backoff.
+async function retryTxn (n, max, operation) {
```

Comment:
This shouldn't be the first snippet of code that a user sees. Here are my suggestions:

1. Just leave it out and mention retrying on, say, our Production Checklist
2. Use a library
3. Make the retry function more terse